extensions/sendbird/v1/actions/deactivateUser/deactivateUser.ts

```diff
@@ -0,0 +1,83 @@
+import { z, ZodError } from 'zod'
+import { fromZodError } from 'zod-validation-error'
+import { type Action } from '@awell-health/extensions-core'
+import { type settings } from '../../../settings'
+import { Category, validate } from '@awell-health/extensions-core'
+import { SettingsValidationSchema } from '../../../settings'
+import { FieldsValidationSchema, fields } from './config'
+import {
+  SendbirdClient,
+  isSendbirdChatError,
+  sendbirdChatErrorToActivityEvent,
+} from '../../client'
+
+export const deactivateUser: Action<typeof fields, typeof settings> = {
+  key: 'deactivateUser',
+  title: 'Deactivate user',
+  description: 'Deactivate user using Chat API.',
+  category: Category.COMMUNICATION,
+  fields,
+  previewable: false,
+  onActivityCreated: async (payload, onComplete, onError) => {
+    try {
+      const {
+        settings: { applicationId, chatApiToken, deskApiToken },
+        fields: { userId, leaveAllGroupChannelsUponDeactivation },
+      } = validate({
+        schema: z.object({
+          settings: SettingsValidationSchema,
+          fields: FieldsValidationSchema,
+        }),
+        payload,
+      })
+
+      const client = new SendbirdClient({
+        applicationId,
+        chatApiToken,
+        deskApiToken,
+      })
+
+      await client.chatApi.updateUser({
+        user_id: userId,
+        is_active: false,
+        leave_all_when_deactivated:
+          leaveAllGroupChannelsUponDeactivation ?? false,
```

Comment:
According to the API specs, the default is `true` so maybe it's wise to follow the defaults from Sendbird as well and set the default to true.